### ISSUE TITLE
Video controls are not read and focused in full screen mode when android talkback is on 

### ISSUE THREAD

**Issue by Jalpesh009** [ionic-team/ionic-framework#23759]
### Prequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [X] v5.x
- [ ] v6.x

### Current Behavior

Video controls are not read and focused in full screen mode when android talkback is on.

### Expected Behavior

Video controls should read and should be focused in full screen mode when talkback is on.

### Steps to Reproduce

1. Turn on talkback.
2. Play any video. Double tap on enter full screen. and tap on any video control.

### Ionic Info

Ionic:

Ionic CLI : 6.16.3 (/usr/local/lib/node_modules/@ionic/cli)
Ionic Framework : @ionic/angular 5.6.9
@angular-devkit/build-angular : 0.1000.8
@angular-devkit/schematics : 10.0.8
@angular/cli : 10.0.8
@ionic/angular-toolkit : 2.3.3

Cordova:

Cordova CLI : 10.0.0
Cordova Platforms : ios 6.2.0
Cordova Plugins : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 2.5.1, (and 16 other plugins)

System:

ios-sim : 8.0.2
NodeJS : v14.17.0 (/usr/local/bin/node)
npm : 6.14.13
OS : macOS Big Sur
Xcode : Xcode 12.5 Build version 12E262```

**Comment by AnkurOO9** [comment]
Please refer below link for code.
https://github.com/hitendraalbiorix/demo

**Comment by willmartindev** [comment]
Hi @AnkurOO9 and @Jalpesh009. I don't think this issue is caused by Ionic Framework, as we do not handle or manipulate video control UI. I recommend reach out to the Ionic forums for additional support on this issue. Thank you!
